src/main/java/com/cgdpd/library/dto/pagination/PaginationCriteria.java

```diff
@@ -0,0 +1,6 @@
+package com.cgdpd.library.dto.pagination;
+
+import java.util.Optional;
+
+public record PaginationCriteria(int pageIndex, int pageSize, Optional<SortParam> sort) {
```

Comment:
Use `@Builder`  and separate each property in one line
And add validations in the constructor to check if values are not invalid